params/config.go

```diff
@@ -575,13 +578,19 @@ type ChainConfig struct {
 	Consortium             *ConsortiumConfig      `json:"consortium,omitempty"`
 	ConsortiumV2Contracts  *ConsortiumV2Contracts `json:"consortiumV2Contracts"`
 	RoninTrustedOrgUpgrade *ContractUpgrade       `json:"roninTrustedOrgUpgrade"`
+	AxieProxyCodeUpgrade   *ContractCodeUpgrade   `json:"axieProxyCodeUpgrade"`
 }
 
 type ContractUpgrade struct {
 	ProxyAddress          common.Address `json:"proxyAddress"`
 	ImplementationAddress common.Address `json:"implementationAddress"`
 }
 
+type ContractCodeUpgrade struct {
+	CodeAddress common.Address `json:"contractAddress"`
+	Code        []byte         `json:"code"`
```

Comment:
https://pkg.go.dev/encoding/json#Marshal

Array and slice values encode as JSON arrays, except that []byte encodes as a base64-encoded string, and a nil slice encodes as the null JSON value.

So in the genesis.json, the code field must be base64 of real code. Is this an expected behavior?

Reply:
You are correct. I have changed the type of Code to string (hex), which will be loaded from genesis file, before being decoded into []byte for statedb process.